src/RokuDeploy.ts

```diff
@@ -1051,3 +1037,28 @@ export interface HttpResponse {
     response: any;
     body: any;
 }
+
+export interface TakeScreenshotOptions {
+    /**
+     * The IP address or hostname of the target Roku device.
+     * @example '192.168.1.21'
+     */
+    host: string;
+
+    /**
+     * The password for logging in to the developer portal on the target Roku device
+     */
+    password: string;
+
+    /**
+     * A full path to the folder where the screenshots should be saved.
+     * Will use the OS temp file system if not supplied
+     */
+    outDir?: string;
+
+    /**
+     * The base filename the zip/pkg file should be given (excluding the extension)
+     * The default format looks something like this: screenshot-YYYY-MM-DD-HH.mm.ss.SSS.<jpg|png>
+     */
+    outFileName?: string;
```

Comment:
I still think this should just be `outFile`. Otherwise, why not call `outDir` `outDirName`? The `dir` and `file` are the names of the things. `outFileName` tells me nothing more than `outFile`?